ui/pages/DataManagement/components/LoadData.jsx

```diff
@@ -97,7 +97,14 @@ const MULTI_DATA_TYPE_CALLSET_PAGE = {
       placeholder: 'gs://',
       ...FILE_PATH_FIELD,
     },
-    ...CALLSET_PAGE_FIELDS,
+    CALLSET_PAGE_FIELDS[0],
```

Comment:
this change seems incorrect to me - why would we be removing fields from the UI as part of this change?